bin/tag-release-reset.js

```diff
@@ -2,6 +2,7 @@
 const commander = require("commander");
 const { extend } = require("lodash");
 const api = require("../src/index.js");
+const workflow = require("../src/workflows/reset");
 
 commander
 	.option("--verbose", "Console additional information")
```

Comment:
is verbose needed in this flow?

Reply:
In order to get our `tag-release --help --verbose` option to work we do, sadly. I thought about just removing it but we had it so I felt I would keep it. I'm not sure anyone EVER looks at that diagram, so might be able to remove it at this point.

Reply:
Probably don't need any of the flags in `reset` but I just left them to be just like all the others. I can remove them.